api/api.go

```diff
@@ -0,0 +1,79 @@
+package api
+
+import (
+	"context"
+
+	"github.com/ONSdigital/dp-search-query/elasticsearch"
+	"github.com/ONSdigital/go-ns/log"
+	"github.com/ONSdigital/go-ns/server"
+	"github.com/gorilla/mux"
+)
+
+var httpServer *server.Server
+
+//SearchQueryAPI provides an API around elasticseach
+type SearchQueryAPI struct {
+	Router *mux.Router
+}
+
+// CreateAndInitialise initiates a new Search Query API
+func CreateAndInitialise(bindAddr, elasticSearchAPIURL string, errorChan chan error) {
+	router := mux.NewRouter()
+	// Setup libraries and handlers
+	elasticsearch.Setup(elasticSearchAPIURL)
+	errSearch := SetupSearch()
+	if errSearch != nil {
+		log.ErrorC("Failed to setup search templates", errSearch, log.Data{})
+	}
+	errData := SetupData()
+	if errData != nil {
+		log.ErrorC("Failed to setup data templates", errData, log.Data{})
```

Comment:
_Copy and paste crime_ with error message. Also I think if this is an error then we would want to `os.Exit` with the appropriate message.